_content/booklets/refreshing-translation-of-the-suttanipata_sujato.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Anthology of Discourses: a Refreshing Translation of the Suttanipāta"
+authors:
+  - sujato
+external_url: "https://suttacentral.net/edition/snp/en/sujato?lang=en"
+drive_links:
+  - "https://drive.google.com/file/d/1-HBJF5CAUcgr1bDCT0wOLT6dbaZ6Cjdp/view?usp=sharing"
```

Comment:
Also include the epub link as the second link: https://drive.google.com/file/d/1yh9p0p12BRsvyjtEluPX1nw82udW2gAD/view?usp=drivesdk